src/documentation/0053-node-difference-dev-prod/index.md

```diff
@@ -33,28 +33,28 @@ Setting the environment to `production` generally ensures that
 
 For example Pug, the templating library used by Express, compiles in debug mode if `NODE_ENV` is not set to `production`. Express views are compiled in every request in development mode, while in production they are cached. There are many more examples.
 
-Express provides configuration hooks specific to the environment, which are automatically called based on the NODE_ENV variable value:
+You can use conditional statements to execute code in different environments:
 
 ```js
-app.configure('development', () => {
+if (app.get('env') === "development") {
   //...
-})
-app.configure('production', () => {
+}
+if (app.get('env') === "production") {
   //...
-})
-app.configure('production', 'staging', () => {
+}
+if(['production', 'staging'].indexOf(app.get('env')) >= 0) {
   //...
 })
 ```
 
-For example you can use this to set different error handlers for different mode:
+For example you can use this to set different error handlers per environment:
```

Comment:
Here we could mention that is an express example, what if we have something like this?

`For example, in an Express app, you can use this to set different error handlers per environment:`